core/src/main/java/com/linecorp/armeria/client/endpoint/PropertiesEndpointGroup.java

```diff
@@ -85,13 +93,42 @@ public static PropertiesEndpointGroup of(ClassLoader classLoader, String resourc
     public static PropertiesEndpointGroup of(ClassLoader classLoader, String resourceName,
                                              String endpointKeyPrefix, int defaultPort) {
         validateDefaultPort(defaultPort);
-        return new PropertiesEndpointGroup(loadEndpoints(
+        final URL resourceUrl = getResourceUrl(
                 requireNonNull(classLoader, "classLoader"),
-                requireNonNull(resourceName, "resourceName"),
+                requireNonNull(resourceName, "fileName"));
+        return new PropertiesEndpointGroup(loadEndpoints(
+                resourceUrl,
                 requireNonNull(endpointKeyPrefix, "endpointKeyPrefix"),
                 defaultPort));
     }
 
+    /**
+     * Creates a new {@link EndpointGroup} instance that loads the host names (or IP address) and the port
+     * numbers of the {@link Endpoint} from the {@code resourceName} resource file. The resource file must
+     * contain at least one property whose name starts with {@code endpointKeyPrefix}:. Reloading property files
+     * for changes is {@code false} by default.
```

Comment:
`Reloading property files ... by default.` could be removed? If you want to emphasize that the other factory methods do not reload, you could write something there, not here.